src/context.ts

```diff
@@ -8,9 +8,9 @@ import {
     matchFilter,
 } from "./filter.ts";
 import {
+    InlineQueryResult,
     type Chat,
     type ChatPermissions,
-    type InlineQueryResult,
```

Comment:
I assume this was a mistake? It should still be a type import.

Reply:
Yep, I must have missed that. I'll update it